src/sentry/tasks/store.py

```diff
@@ -63,6 +65,12 @@ def _do_preprocess_event(cache_key, data, start_time, event_id, process_event):
     })
 
     if should_process(data):
+        # save another version of data for some projects to generate
+        # preprocessing hash that won't be modified by pipeline
+        if data.get('event_id') and int(project) % 5 == 0:
+            raw_cache_key = get_raw_cache_key(project, data['event_id'])
```

Comment:
I don't know as the local variable assignment for `raw_cache_key` here is all that helpful

Reply:
good call, i possibly changed/copied from a longer line or something where it made more sense. will change